src/jquery.uls.lcd.js

```diff
@@ -363,22 +362,20 @@
 		},
 
 		noResults: function () {
-			var $suggestions;
-			this.$noResults.removeClass( 'hide' );
-			this.$noResults.siblings( '.uls-lcd-region-section' ).addClass( 'hide' );
+			var $suggestions, $noResults;
 
-			// Only build the data once
```

Comment:
I had no idea why this check was done. The quickList is cached anyway. Also this was causing the issue: When I search for something and get no results everything fine, then I do a search and get some result. Then search for some non-existing language-The no-results screen then won't have suggestions.

I just removed this check